src/configs/ssh_agent.rs

```diff
@@ -0,0 +1,28 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(feature = "config-schema", derive(schemars::JsonSchema))]
+#[serde(default)]
+pub struct SshAgentConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub symbol_multi: &'a str,
+    pub none_symbol: &'a str,
+    pub style: &'a str,
+    pub none_style: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> Default for SshAgentConfig<'a> {
+    fn default() -> Self {
+        Self {
+            format: "[ssh $symbol]($style)",
+            symbol: "🔑 ",
+            symbol_multi: "🔑➕ ",
+            none_symbol: "❌ ",
```

Comment:
`symbol_none`, to fit the other two names?

Reply:
For some reason the `Format + Docs Workflow / Dprint [Docs Formatter] (pull_request) ` worklow is now failing but I'm unable to reproduce the issue locally.